src/components/Header/Header.js

```diff
@@ -7,14 +7,14 @@ import SiteLogo from "./subcomponents/SiteLogo";
 import SiteText from "../../icons/SiteText";
 import LandingForm from "../../pages/Landing/subcomponents/LandingForm";
 import UserHeaderContainer from "./subcomponents/UserHeaderContainer";
-import { resolveFieldValueOrError } from "../../../node_modules/graphql/execution/execute";
 
 export default function Header({ userLoggedIn, avatarIndex, color }) {
   let [showHamburgerDropdown, handleHamburgerClick] = useState(false);
-  let [formIsOpen, setFormIsOpen] = useState(userLoggedIn || window.innerWidth < 1200 ? false : true);
+  let [formIsOpen, setFormIsOpen] = useState(
+    userLoggedIn || window.innerWidth < 1200 ? false : true
+  );
 
-  function filler ({som}) {
-  }
+  function filler() {}
```

Comment:
if we're not using this for anything, may make sense to remove or just pass an anonymous blank function to `handleHamburgerClick`. E.g., `handleHamburgerClick={() => {}}`;

If we're just using as a placeholder blank/empty function call for now, I think that's fine 👍